content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+```
+
+Instead, use the following command:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias maprca -cacerts -storepass <cacerts_truststore>
+```
+
+If you use the Installer to install HPE Ezmeral Data Fabric on a fresh OS, then Installer will automatically install JDK 11, then "-storepass" password is "changeit".
+
+There is another place in the documentation where keytool is used, the command is as follows:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias mosscert -keystore ${JAVA_HOME}/lib/security/cacerts -storepass changeit
+```
+
+You don't need to execute `keytool` twice to import the same cacert certificate file.
+
+I suggest you change "maprca" to something more recognizable.
+
+For example, if you named the cluster "edf-cluster-a.mycompany.com" when you installed the HPE Ezmeral Data Fabric cluster, then you can use the following keytool command:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias edf-clustera-ca -keystore ${JAVA_HOME}/lib/security/cacerts - storepass <cacerts_truststore>
+```
+
+The file - /opt/mapr/conf/ca/chain-ca.pem, is a self-signed TLS certificate file created by the Installer when configuring wire-level encryption for the cluster.
+
+Since it's a self-signed TLS certificate, the client (application, or your browser) will not be able to trust the TLS certificate of the server when accessing the HPE Ezmeral Data Fabric server.
+
+This is because the CA certificate used by the self-signed TLS certificate is not publicly trusted.
+
+In any scenario where a self-signed TLS certificate is used, you need to import the self-signed CA certificate into the OS system.
+
+You can use the following command:
+
+```shell
+keytool -list -v -cacerts
+```
+
+to look at things like CA certificates in your JVM.
+
+You'll find something like the following:
+
+```dart
+Alias name: digicertassuredidrootca
+Creation date: Feb 2, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+Issuer: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+...
+```
+
+This is a recognized CA certificate issued by Digicert.
+
+So, after you use `keytool` to import /opt/mapr/conf/ca/chain-ca.pem into the JVM, use keytool -list -v -cacerts to see something similar to the following:
+
+```dart
+Alias name: edf-clustera-ca
+Creation date: Feb 16, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=MapR Engineering Signing CA, OU=MapR Engineering Signing CA, O=MapR, DC=hpecorp, DC=net
+Issuer: CN=MapR Engineering Root CA, OU=MapR Engineering Root CA, O=MapR, DC=hpecorp, DC=net
+...
+```
+
+#### 2. The documentation describes several client software that you can use to interact with the HPE Ezmeral Object Store. I would like to add the following clarification:
```

Comment:
add "options" or "packages" after "...several client software..."
Because you're using a header font for #1 and #2, the formatting is a little weird. The sentences don't go all the way to the right before they wrap. You might want to consider doing this differently. Maybe you don't need full sentences in those numbered bullets. It looks like you're using them to create sections in the article. Maybe shorten them to:

1. About "keytool -noprompt -importcert" command
2. About client software options
And then take a little of what you have in the rest of those bullets currently and put it in as a sentence below

Reply:
Yes, I'll make the numbered bullet header shorter and add description below.